technology/typemapping/store/jdbc/src/main/java/com/github/mizool/technology/typemapping/store/jdbc/JdbcValueLoadStrategyResolver.java

```diff
@@ -0,0 +1,44 @@
+package com.github.mizool.technology.typemapping.store.jdbc;
+
+import java.sql.Types;
+import java.util.Map;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+
+import com.github.mizool.core.exception.UnprocessableEntityException;
+import com.google.common.collect.ImmutableMap;
+
+public class JdbcValueLoadStrategyResolver
+{
+    private final Map<Integer, JdbcValueLoadStrategy> strategies;
+
+    @Inject
+    protected JdbcValueLoadStrategyResolver(Instance<JdbcValueLoadStrategy> strategies)
+    {
+        ImmutableMap.Builder<Integer, JdbcValueLoadStrategy> builder  = ImmutableMap.builder();
+
+        Stream<JdbcValueLoadStrategy> jdbcValueLoadStrategyStream
```

Comment:
`javax.enterprise.inject.Instance` implements `java.lang.Iterable` which in turn offers a `forEach` method.

Using a stream was the correct approach when we used a `collect` call to build the `Map`. Since that has changed, remove the `Stream` altogether and just use `strategies.forEach(jdbcValueLoadStrategy -> ...);`